Detections/ASimNetworkSessions/PossibleBeaconingActivity.yaml

```diff
@@ -0,0 +1,65 @@
+id: fcb9d75c-c3c1-4910-8697-f136bfef2363
```

Comment:
All the comments from the previous detection I reviewed: 
- Use custom field, custom alert details.
- Parent Tags